src/dash-table/handlers/cellEvents.ts

```diff
@@ -1,7 +1,7 @@
 import * as R from 'ramda';
 import { SelectedCells, ICellFactoryProps } from 'dash-table/components/Table/props';
 import isActive from 'dash-table/derived/cell/isActive';
-import reconcile from 'dash-table/reconcile';
+import reconcile from 'dash-table/type/reconcile';
```

Comment:
Restructured the /reconcile (coerce, validate) folder to a more general /type folder that contains a variety of operations on the types (now coerce, validate, format)